assets/src/components/mapPage.tsx

```diff
@@ -205,6 +205,9 @@ const Selection = ({
 }
 
 const MapPage = (): ReactElement<HTMLDivElement> => {
+  const [followerInitializeState, setFollowerInitializeState] =
+    useState<boolean>(false)
```

Comment:
One suggestion: drop the "state" from the name.

As for the rest of the name, how about something like `followerDefaultZoomOnInitialize` or something like that? Or the opposite of that if I have the logic backwards.

Reply:
What do you think of `followerHasRun` or `followerHasHadAChanceToRun`?

Reply:
Hmm, I see what you're getting at... it sort of approaches this functionality from a different angle (basically it's expressing the condition causes the behavior to change rather than what the behavior is). I'm trying to decide if I feel like that helps or hurts when it comes to clarity. 😅 In any event since "run" is a bit of an overloaded term in Skate I'd prefer `followerHasHadAChanceToRun` out of those two options.

Reply:
Hah, I didn't even think about how overloaded "run" is at first. I agree the other is better.

I think it reads better `vehicleUseCurrentZoom={followerHasHadAChanceToRun === true}` "vehicle use current zoom if follower has had a chance to run"
but maybe 
`vehicleUseCurrentZoom={followerNeedsToInitializeZoom === false}` would be better?
Because I changed it so that the zoom control state is reset when a item is selected from search (so you don't stay super zoomed out)

Reply:
Oh I like `followerNeedsToInitializeZoom`!

Reply:
Okay I changed it once more, let me know if you'd prefer `NeedsTo` instead of `Should`, I was uncertain what would better convey the logic. `Should` seemed better but I'm unsure.

bd56b922151fcddb6e07258e39910167538dc0f5

Reply:
I'm neutral so if you prefer `Should` that sounds good to me. 👍